VAMobile/src/screens/HomeScreen/VeteranStatusScreen/VeteranStatusScreen.tsx

```diff
@@ -0,0 +1,143 @@
+import { DisabilityRatingState, MilitaryServiceState, PersonalInformationState } from 'store/slices'
+import { HomeStackParamList } from '../HomeStackScreens'
+import { NAMESPACE } from 'constants/namespaces'
+import { RootState } from 'store'
+import { getBirthDate } from '../ProfileScreen/PersonalInformationScreen/PersonalInformationScreen'
+import { useHasMilitaryInformationAccess } from 'utils/authorizationHooks'
+import { useTheme } from 'utils/hooks'
+// import PhotoUpload from 'components/PhotoUpload'
+
+type VeteranStatusScreenProps = StackScreenProps<HomeStackParamList, 'VeteranStatus'>
+
+const VeteranStatusScreen: FC<VeteranStatusScreenProps> = () => {
+  const { profile } = useSelector<RootState, PersonalInformationState>((state) => state.personalInformation)
+  const { serviceHistory, mostRecentBranch } = useSelector<RootState, MilitaryServiceState>((s) => s.militaryService)
+  const { ratingData } = useSelector<RootState, DisabilityRatingState>((state) => state.disabilityRating)
+  const accessToMilitaryInfo = useHasMilitaryInformationAccess()
+  const theme = useTheme()
+  const { t } = useTranslation(NAMESPACE.COMMON)
+
+  const name = (): string => {
+    return profile?.fullName || ''
+  }
+
+  const ratingPercent = ratingData?.combinedDisabilityRating
+  const ratingIsDefined = ratingPercent !== undefined && ratingPercent !== null
+  const combinedPercentText = ratingIsDefined ? t('disabilityRating.combinePercent', { combinedPercent: ratingPercent }) : undefined
+
+  const getPeriodOfService: React.ReactNode = map(serviceHistory, (service: ServiceData) => {
+    const branch = t('militaryInformation.branch', { branch: service.branchOfService })
+    return (
+      <Box>
+        <Box display="flex" flexDirection="row" alignItems="center" mt={theme.dimensions.condensedMarginBetween}>
+          <TextView variant="MobileBody" color="primaryContrast">
+            {branch}
+          </TextView>
+        </Box>
+        <Box>
+          <TextView variant="HelperText" color="primaryContrast" mb={theme.dimensions.condensedMarginBetween}>
+            {t('militaryInformation.history', { begin: service.formattedBeginDate, end: service.formattedEndDate })}
+          </TextView>
+        </Box>
+      </Box>
+    )
+  })
+
+  const branch = mostRecentBranch || ''
+
+  const boxProps: BoxProps = {
+    minHeight: 81,
+    borderRadius: 6,
+    pt: theme.dimensions.cardPadding,
+    pb: theme.dimensions.cardPadding,
+    backgroundColor: theme.colors.background.veteranStatus as BackgroundVariant,
+    borderTopWidth: theme.dimensions.borderWidth,
+    borderColor: theme.colors.border.veteranStatus as BorderColorVariant,
+    borderStyle: 'solid',
+  }
+
+  const getBranchSeal = (): React.ReactNode => {
+    const dimensions = {
+      width: 34,
+      height: 34,
+    }
+
+    switch (branch) {
+      case BranchesOfServiceConstants.AirForce:
+        return <VAIcon testID="United States Air Force" name="AirForce" {...dimensions} />
```

Comment:
For screen readers, consider giving them something like "US Air Force Seal", or else hide this from screen readers entirely (probably everyone thought of this already, but just in case)

Reply:
Outside of scope here.